src/components/Team/index.tsx

```diff
@@ -0,0 +1,170 @@
+import TwitterMUIIcon from '@mui/icons-material/Twitter';
+import { Avatar, Box, Divider, styled, Typography } from '@mui/material';
+import { SectionTitle } from '../Commons/SectionTitle';
+import { GithubCatIcon } from '../Commons/SVGs/GithubCat';
+
+const MemberBox = styled(Box)(({ theme }) => ({
+  justifyContent: 'flex-start',
+  alignItems: 'center',
+  display: 'flex',
+  flexDirection: 'column',
+  rowGap: theme.spacing(2),
+  flex: '1 1 20%',
+}));
+
+const MemberName = styled(Typography)(({ theme }) => ({
+  color: theme.palette.common.white,
+  fontWeight: '800',
+  fontSize: '18px',
+  textAlign: 'center',
+  [theme.breakpoints.down('md')]: {
+    width: '100%',
+    fontSize: '16px',
+  },
+}));
+
+const MemberDescription = styled(Typography)(({ theme }) => ({
+  color: theme.palette.common.white,
+  whiteSpace: 'break-spaces',
+  fontWeight: '600',
+  fontSize: '16px',
+  textAlign: 'center',
+  [theme.breakpoints.down('md')]: {
+    width: '100%',
+    fontSize: '14px',
+  },
+}));
+
+type TeamMember = {
```

Comment:
I also like this, I would add the url

Reply:
contactType as you proposed bellow should be a enum either Twitter or Github, depending on that the icon should change. If it's fast to update that, great, otherwise I would only focus on add href to the members social media

Reply:
Another thing, but this is for fixing another time (I created an issue #19) the info should be parametrized and not hardcoded